data/transport missions.txt

```diff
@@ -3469,3 +3469,114 @@ mission "Inheritance Redirection"
+			label "hello too"
+			`She says, "My name is Vanessa Oxygenpocket and I am part of a team that is designing the new Hauler VI transport ship series here at Southbound Shipyards."`
+			choice
+				`	"'Oxygenpocket' is an uncommon name."`
+				`	"And what help do you need from me?"`
+					goto "help request"
+			`	Vanessa looks surprised and then responds, "Well, it might sound uncommon to people from a planet with a breathable atmosphere, but it is actually similar to Vanderberg, which means 'from the mountain', or Langley, which means 'woodland'.`
+			`	"Oxygenpockets were as much a feature on my home-planet as mountains and woodlands are on more habitable planets."`
+			label "help request"
+			`	Vanessa pauses for a moment and then continues, "We're testing the VI's cargo systems using a Hauler III. Normally our own department would escort such prototypes, but there is a mole in our organization and we have very strong competition from Syndicated Shipyards. I need you to help us travel to <destination> to deliver sushi for a formal dinner. You will be paid for your expenses. Will you help us?"`
+			choice
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it remains in orbit.`
+			`There is no response to your hails by radio. You need to take off and investigate.`
+
+mission "Hauler VI cargo prototype 2"
+	name "Escort Hauler prototype to student party"
+	description "Escort the Hauler prototype to <destination> to deliver cooked fish dishes to a streetfood festival."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Hauler VI cargo prototype 1: done"	
+	npc accompany
+		government "Merchant"
+		personality escort timid
+		ship "Hauler III" "SB-Labs Testament"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel, "Why were you trying to contact me over radio! Does everyone need to know that we are flying around here?"`
+			`She adds, "The cooling failed and the sushi nearly cooked, let's go to <destination> to deliver it as fish dishes to a streetfood festival."`
+	on visit
+		dialog `You land on <destination>, but the Hauler prototype didn't catch up with you yet.`
+	on complete
+		dialog
+			`This time the Hauler prototype appears to be landing right behind you, but the ship flies away at full thrust just before touching the landing pad.`
+			`You receive a short text-message over the short-range inter-ship communiation channel, "The sushi overcooked! We cannot be seen near the festival or near the shipyard with this overcooked sushi!"`
```

Comment:
```suggestion
			`You receive a text message over the short-range inter-ship communication channel: "The sushi overcooked! We cannot be seen near the festival or near the shipyard with this overcooked sushi!"`
```
Why do they care? I thought the point was to test out the ship. Why so much fuss about sushi?

Reply:
The test was for the ships refrigeration/cooling systems. If the transported food becomes overcooked and burned, then the test obviously failed.

Reply:
And Vanessa doesn't like her colleagues from Southbound Shipyards or anybody else to see how badly the test failed. So that's why she wants to leave in silence.